### ISSUE TITLE
Popups don't gain focus when opened

### ISSUE THREAD

**Issue by bleistivt** [vanilla/vanilla#8637]
Popups are hardly accessible using the keyboard only: tab order is broken. When a popup is opened, focus is still on the button that opened it and using the tab key will only send you further down the page. This applies to both the old (+Baseline) and new style (keystone) popups (test with the signin popup on vf.org).

The popups used in the dashboard do not have this problem, however they don't return focus to the element, that opened them, when being closed using ESC even though this has been checked here:
https://github.com/vanilla/vanilla/issues/6806

To Reproduce navigate to any link/button that opens a popup using the keayboard only.

**Comment by charrondev** [comment]
Dismissing as this is still a valid issue. Our latest generation popup (`<Modal />`) handles the focused correctly according to the WCAG spec, but it is not in use everywhere.

**Comment by slafleche** [comment]
Here's the spec: https://www.w3.org/TR/wai-aria-practices/examples/dialog-modal/dialog.html
